ami_config.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Include ami_config role
   hosts: ami_instances
-  strategy: debug
+  #strategy: debug
```

Comment:
@sabre1041 this is why you were being dropped to the prompt when the task failed.